cookbooks/ambari/recipes/ambari_views_setup.rb

```diff
@@ -0,0 +1,40 @@
+#
+# Cookbook Name:: ambari
+# Recipe:: ambari_views_setup
+#
+# Copyright (c) 2016 The Authors, All Rights Reserved.
+
+
+http_request 'Polling Ambari WebUi' do
+  url "#{node['ambari']['ambari_server_base_url']}"
+  retries 24
```

Comment:
Any reason to do 24 retries? It seems like 120 seconds is a long time to wait...

Reply:
I just chose a random number. Usually, ambari-server comes up within few seconds but I set the limit to 2 minutes just to give ample time to start ambari server and its components.